src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -161,7 +168,8 @@ describe('HmctsGlobalHeaderComponent', () => {
       href: '',
       active: false
     }];
-    component.ngOnChanges(changesMock);
+    await component.ngOnChanges(changesMock);
```

Comment:
@johnbenjamin-hmcts - similar to above, I thought @ronaldmansveld got rid of the `await` because he was no longer using `async()` and is calling `done()` at the end.

Reply:
Thank you @Daniel-Lam - I have now reverted the changes.